integration_tests/e2e/calendar-overtime.cy.ts

```diff
@@ -19,7 +19,7 @@ context('A staff member can view their overtime calendar', () => {
     })
     cy.login()
 
-    Page.verifyOnPageTitle(CalendarPage, format(Date.now(), 'MMMM yyyy'))
+    Page.verifyOnPageTitle(CalendarPage, format(new Date(), 'MMMM yyyy'))
```

Comment:
hadn't realised that `Date.now` doesn't return a `Date` object, instead returns number of millis since 1970.